posix.ipkg

```diff
@@ -3,6 +3,14 @@ package posix
 sourcedir = src
 modules = System.Posix.Directory
         , System.Posix.Time
+	, System.Posix.Signal
+	, System.Posix.Syslog
+	, System.Posix.Test.Signal
+	, System.Posix.Test.Syslog
```

Comment:
I expect the parser must be whitespace oblivious. Anyway, I committed the change to only using spaces.